diagnostics/seaice_suite/seaice_suite.html

```diff
@@ -0,0 +1,89 @@
+<title>Sea Ice Suite</title>
+<img src="../mdtf_diag_banner.png">
+<h3>Sea Ice Suite</h3>
+<p> This POD calculates maps of the sea ice concentration mean, trend,
+standard deviation, and one-lag correlation by month. Sea ice
+concentration from passive microwave satellite from HadISSTv1.1 is
```

Comment:
Consider adding a hyperlink to the reference dataset.

Reply:
@jkrasting Yes, given that there are several smaller commits interspersed with updates to the develop branch from NOAA-GFDL.
@lettie-roach Please try rebasing your branch onto the NOAA-GFDL/develop branch after resolving any outstanding issues and code changes requested by @jkrasting, @tsjackson-noaa, and me (see [here](https://github.com/NOAA-GFDL/MDTF-diagnostics/blob/develop/doc/sphinx/dev_git_intro.rst#updating-your-feature-branch-by-rebasing-it-onto-the-develop-branch-preferred-method) for instructions). If there are issues, or something in the instructions is not clear, leave a response in the comments.

Reply:
@wrongkindofdoctor should I do this now, or do I need to wait for more reviews?

I looked at how this would work. It comes up with 'Rebase ea48ba3..92a073b onto ea48ba3' - is that correct? Should I 'squash' anything?

Reply:
@lettie-roach It looks like the develop branch on your fork is one commit behind NOAA-GFDL/develop, so you should update your remote and local develop branches first. Once  @tsjackson-noaa has approved the PR, proceed with the rebase. Squashing your commits into one will consolidate the log, so give it a try. 

Based on your commit history, I recommend squashing all commits starting with your most recent commit [727c2ec](https://github.com/lettie-roach/MDTF-diagnostics/commit/727c2ec0eb7956c5da0461257ed15a2abbdb91a1) onto the oldest commit on your branch before you last pulled in changes from NOAA-GFDL [b8a05db](https://github.com/lettie-roach/MDTF-diagnostics/commit/b8a05db0ade7da8518460626c2dd76021dc410b9). After updating your remote develop branch and switching to the pod/seaicesuite branch, the Vim editor that opens after calling `git rebase -i develop` will display a commit log. Next, change all of the commits **older than** the commit that you want to "squash onto" from "pick" to "squash" (note that you may have to scroll through the history to access all of the commits you want to squash).
```
pick f0661e6 (the commit corresponding to the last NOAA-GFDL update you merged into your branch)
pick b8a05db (oldest commit on your branch after the last NOAA-GFDL update)
squash 4c1ddce 
(squash the rest of the commits in between)
squash 727c2ec (most recent commit on your branch)
```